mkdocs.yml

```diff
@@ -0,0 +1,104 @@
+site_name: ReactiveUI Documentation
+site_url: http://docs.reactiveui.net/
+site_url: /
+site_description: A MVVM framework that integrates with the Reactive Extensions for .NET to create elegant, testable User Interfaces that run on any mobile or desktop platform. Supports Xamarin.iOS, Xamarin.Android, Xamarin.Mac, WPF, Windows Forms, Windows Phone 8 and Windows Store apps.
+site_favicon: img/favicon.png
+
+dev_addr: '0.0.0.0:8000'
```

Comment:
I'll keep this here as part of self-documenting of outcomes. Signifies to contributors where mkdocs will be available from and shows that it can be adjusted.